dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/program/hibernate/Event.hbm.xml

```diff
@@ -51,8 +51,8 @@
     <many-to-one name="organisationUnit" class="org.hisp.dhis.organisationunit.OrganisationUnit" column="organisationunitid"
       foreign-key="fk_programstageinstance_organisationunitid" index="programstageinstance_organisationunitid" />
 
-    <list name="messageConversations" table="programstageinstance_messageconversation">
-      <key column="programstageinstanceid" />
+    <list name="messageConversations" table="event_messageconversation">
```

Comment:
Is `event_messageconversation` going against our convention of all lowercase with no separators like `eventcomments`? If so should we remove the `_` as we already rename the table?

Reply:
this table and also `programinstance_messageconversation` I understand we'll be removed soon because no longer in use so we can leave it like this